app/src/views/Privacy.vue

```diff
@@ -245,6 +245,53 @@
 
         <h2>3. Datenerfassung auf unserer Website</h2>
 
+        <h3>Nutzungsstatistiken</h3>
+
+        <p>
+          Diese Webseite verwendet Matomo, eine Open Source, selbstgehostete
+          Software um anonyme Nutzungsdaten für diese Webseite zu sammeln. Die
+          Daten zum Verhalten der Besucher werden gesammelt um eventuelle
+          Probleme wie nicht gefundene Seiten, Suchmaschinenprobleme oder
+          unbeliebte Seiten herauszufinden. Sobald die Daten (Anzahl der
+          Besucher die Fehlerseiten oder nur eine Seite sehen, usw.) verarbeitet
+          werden, erzeugt Matomo Berichte für die Webseitenbetreiber, damit
+          diese darauf reagieren können. (Layoutveränderungen, neue Inhalte,
+          usw.). Die Daten werden lediglich auf den Servern von
+          einander-helfen.org gespeichert und nicht an Dritte weitergegeben.
+        </p>
+
+        <p>Matomo verarbeitet die folgenden Daten:</p>
+        <ul>
+          <li>
+            Anonymisierte IP-Adressen indem die letzten 2 bytes entfernt werden
+            (also 198.51.0.0 anstatt 198.51.100.54)
+          </li>
+          <li>Sprache des Nutzers anhand der Browsereinstellung</li>
+          <li>Datum und Uhrzeit</li>
+          <li>Titel der aufgerufenen Seite</li>
+          <li>URL der aufgerufenen Seite</li>
+          <li>URL der vorhergehenden Seite (sofern diese das erlaubt)</li>
+          <li>Bildschirmauflösung</li>
+          <li>Lokale Zeit</li>
+          <li>Externe Links</li>
+          <li>Dauer des Seitenaufbaus</li>
+          <li>User Agent des Browsers</li>
+        </ul>
+
+        <br />
+        <p v-if="!optOutIframeExists()">
```

Comment:
dont use a function here, use a data attribute which is set by mounted or some other lifecycle hook.